interfaces/builtin/microstack_support.go

```diff
@@ -255,11 +242,11 @@ func init() {
 		summary:                  microStackSupportSummary,
 		implicitOnCore:           true,
 		implicitOnClassic:        true,
+		controlsDeviceCgroup:     true,
```

Comment:
should we add a TODO somewhere about this implying Deletage=true?

Reply:
Done

Reply:
@mardy you added a comment about this implication but not in the vein that I think Samuele wanted, I think we want something in spec.go around the functions for controlsCgroup etc or perhaps in common.go

Reply:
ah well I just pushed this anyways so no need to make changes